kit/src/components/message/mod.rs

```diff
@@ -88,11 +87,15 @@ pub struct Props<'a> {
 }
 
 fn wrap_links_with_a_tags(text: &str) -> String {
-    let re = Regex::new(r#"(?i)\b((?:https?://|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,4}/)(?:[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:'".,<>?«»“”‘’]))"#).unwrap();
+    let re = regex::Regex::new(r#"(?i)\b((?:(?:https?://|www\.)[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:'".,<>?«»“”‘’]))"#).unwrap();
 
     re.replace_all(text, |caps: &regex::Captures| {
         let url = caps.get(0).unwrap().as_str();
-        format!("<a href=\"{}\">{}</a>", url, url)
+        if url.starts_with("www.") {
+            format!("<a href=\"http://{}\">{}</a>", url, url)
```

Comment:
we should force https here and not rely on forced redirects on whatever site they are on

Reply:
done 🎉